dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -472,5 +472,9 @@ variables.in.id.not.supported=CWWKG0104W: The ID attribute with value {1} for th
 variables.in.id.not.supported.explanation: Variable values are not allowed in ID attributes. All variables in ID attributes will be treated as literal strings.  
 variables.in.id.not.supported.useraction: Remove the variable from the value of the ID attribute.
 
+error.root.must.be.server=CWWKG0105E: The configuration document {0} must begin with the element '{1}'.
```

Comment:
**change**
The configuration document {0} must begin with the element '{1}'.
**to**
The {0} configuration document must begin with the '{1}' element.

why is one offset with brackets and straight  quote? '{1}' Are both needed?

Reply:
The only reason for the quote on {1} is that without it the phrase sounds like a concept. It's better with the phrasing you suggested, but with either "server element" or "element server" it sounds like a particular server or type of server. I can remove it if needed, though.

Reply:
Ok- I see your point. The concern is that using quotes can flag an accessibility violation in some sources. I'm not sure if that would apply in this case but it's something we watch out for. I don't know where the output is expected but in a markdown source backtics (`) around `server` could do the job. [Brackets] are also a possibility.

Reply:
We use brackets elsewhere for config messages, so I'll go with that.